pkg/debug/spec.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/sh
 #shellcheck disable=SC2039
-# This script creates an initial hardware model file
-# The output is conservative when it comes to assignment groups in that
-# it does not form any assignment groups but merely checks whether there are
-# multiple functions on the same controller, and in that case the assignment
-# group is set to empty.
+# This script creates an initial hardware model file.
+# It should be run on KVM and without having already made some adapters be
```

Comment:
Maybe a good idea to check for /dev/kvm and issue a warning if it isn't there @eriknordmark

Reply:
@rvs I guess one could also run it on e.g., ubuntu on the metal to get this data? So perhaps check for Xen and fail in that case?

Reply:
Personally, I'd prefer a big fat warning still -- simply b/c this tools is meant to produce enough data to get the human going, but it isn't really meant to be 100% fully automates. And some data (even in case of Xen) is better than no data. You may also add some scary looking GROUP names if it runs under Xen to be output into the model.

Reply:
Did that by putting the warning in the assigngrp in the output if running on Xen.